srcpkgs/hsetroot/template

```diff
@@ -1,18 +1,24 @@
 # Template file for 'hsetroot'
 pkgname=hsetroot
-version=1.0.2
-revision=9
-wrksrc="hsetroot-${version}.orig"
-build_style=gnu-configure
+version=1.0.5
+revision=10
+wrksrc="hsetroot-${version}"
+build_style=gnu-makefile
 hostmakedepends="automake libtool pkg-config"
-makedepends="libX11-devel imlib2-devel"
+makedepends="libX11-devel imlib2-devel libXinerama-devel"
+depends="imlib2"
 short_desc="Tool which changes X server wallpapers"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Felipe Nogueira <contato.fnog@gmail.com>"
 license="GPL-2.0-only"
-homepage="https://directory.fsf.org/wiki/Hsetroot"
-distfiles="${DEBIAN_SITE}/main/h/hsetroot/hsetroot_${version}.orig.tar.gz"
-checksum=597fe1175e590d54d0927095ce361c86ddef15ebc70a96c4bf509f6cb9165723
+homepage="https://github.com/himdel/hsetroot"
+distfiles="https://github.com/himdel/hsetroot/archive/${version}.tar.gz"
+checksum=cff5f83b85a21e0c5c8c8eecbd552090bb75d8ac59e27b0bae48046f9cb5a44a
 
-pre_configure() {
-	autoreconf -if
+do_build() {
+	make
+}
+
+do_install() {
```

Comment:
This should also be left to the build style, the makefile has an install target.